File: solutions/de.vimotest.sandbox/source_gen/de/vimotest/sandbox/dependencies/calls/VerifyDependencyCallViewModelTestSetupImpl.cs
/// <filename>
///     VerifyDependencyCallViewModelTestSetupImpl.cs
/// </filename>
using dependencies.calls;
using dependencies;

namespace dependencies.calls
{
    internal class VerifyDependencyCallViewModelTestSetupImpl : dependencies.calls.VerifyDependencyCallViewModelTestSetup
    {
        public override void Init()
        {
        }

        public override dependencies.calls.VerifyDependencyCallViewModel BuildSut()
        {
            return null;
        }

        public override dependencies.LoggerSpy getLogger()
        {
            return null;
        }

        public override dependencies.calls.VerifyDependencyCallViewModel getSimpleDependency_ViewModel()
        {
            return null;
        }
    }
}

File: solutions/de.vimotest.sandbox/source_gen/de/vimotest/sandbox/dependencies/calls/VerifyDependencyCallViewModelTestSetupImpl.cpp
#include "VerifyDependencyCallViewModelTestSetupImpl.h"
#include <memory>
#include "VerifyDependencyCallViewModel.h"
#include "dependencies/LoggerSpy.h"

namespace dependencies::calls
{
  void VerifyDependencyCallViewModelTestSetupImpl::Init()
  {
  }
  std::shared_ptr<VerifyDependencyCallViewModel> VerifyDependencyCallViewModelTestSetupImpl::BuildSut()
  {
    return nullptr;
  }
  std::shared_ptr<LoggerSpy> VerifyDependencyCallViewModelTestSetupImpl::getLogger()
  {
    return nullptr;
  }
  std::shared_ptr<VerifyDependencyCallViewModel> VerifyDependencyCallViewModelTestSetupImpl::getSimpleDependency_ViewModel()
  {
    return nullptr;
  }
}
